.github/scripts/check-engines-cli.sh

```diff
@@ -92,13 +88,8 @@ else
 fi
 
 
-echo "--- yarn prisma -v ---"
-yarn -s prisma -v
-echo "--- ls -lh node_modules/@prisma/engines/ ---"
-ls -lh node_modules/@prisma/engines/
-echo "--- ls -lh node_modules/prisma/ ---"
-ls -lh node_modules/prisma/
```

Comment:
This used to be useful to get an overview of the files. 
Not possible any more with pnpms placing of node_modules I guess?

Reply:
I can restore it

Reply:
Not really necessary I think. Maybe commented out for when we need to debug things in this script.